packages/express/package.json

```diff
@@ -68,6 +68,9 @@
     "tsup": "*",
     "typescript": "*"
   },
+  "peerDependencies": {
+    "express": "^4.17.0 || ^5.0.0"
```

Comment:
we support v5 out-of-the-box actually but we want to set a more stable minimum v4 version to prevent any breaking change in earlier versions

Reply:
`@clerk/express` is still  v0. Are we doing v1 after this PR, or we wait for Express 5 to be cut as latest and then do v1 ?

Reply:
Lets do v1 now unless we know that express v5 is coming out *soon*

Reply:
For reference, here's the pending items before they tag v5 as latest:

<img width="816" alt="Screenshot 2024-09-23 at 7 52 45 AM" src="https://github.com/user-attachments/assets/516d4fc7-1029-4b32-8cf9-cb8f4b1d9d84">

Mostly docs update, so I think `latest` tag will come soon. Let's go v1 major?